src/elements/frame-router.ts

```diff
@@ -129,11 +129,11 @@ export default class FrameRouterElement extends HTMLElement {
    */
   public attributeChangedCallback(
     name: string,
-    oldValue: string,
-    newValue: string
+    oldValue: string | null,
+    newValue: string | null
   ) {
     if (name === ROUTE_ATTR && oldValue !== newValue) {
-      this.changeRoute(newValue);
+      this.changeRoute(String(newValue || ''));
```

Comment:
You don't need the `String(...)` constructor here.  I probably would have let the null value cascade through, but this will also work and isn't hard to change if we need to in the future.